ci/script.sh

```diff
@@ -29,7 +29,7 @@ main() {
     esac
 
     if [ $TARGET = x86_64-unknown-linux-gnu ]; then
-        ./check-blobs.sh
+        cargo xtask check-blobs
```

Comment:
Hmm, this might break between nightly updates when the version changes. That probably means we'll have to pin the nightly version to use for the blobs, which is probably a good idea anyways.

Reply:
Yes, I agree, we should definitely pin a version so we don't risk all kinds of funky random issues.

Reply:
Added an `asm-toolchain` file to pin the nightly version. `cargo xtask` will install it automatically.